tutorials/backend/hasura-v3-supergraph/tutorial-site/config.js

```diff
@@ -0,0 +1,87 @@
+const config = {
+  gatsby: {
+    pathPrefix: '/learn/graphql/hasura-v3-supergraph',
+    siteUrl: 'https://hasura.io',
+    gaTrackingId: 'GTM-WBBW2LN',
+    trailingSlash: true,
+  },
+  header: {
+    logo: 'https://graphql-engine-cdn.hasura.io/img/hasura_icon_black.svg',
+    logoLink: 'https://hasura.io/learn/',
+    title:
+      "<a href='https://hasura.io/learn/'>learn</a><img src='https://graphql-engine-cdn.hasura.io/learn-hasura/assets/chevron-right.svg' alt='Chevron Right' /><a href='https://hasura.io/learn/graphql/hasura/hasura-v3-supergraph/introduction/'>hasura</a>",
+    githubUrl: 'https://github.com/hasura/learn-graphql',
+    helpUrl: 'https://discord.com/invite/hasura',
+    tweetText:
+      'Check out this Introduction to the Hasura v3 Supergraph modeling course for architects and backend engineers by @HasuraHQ https://hasura.io/learn/graphql/hasura/hasura-v3-supergraph/introduction/',
+    links: [
+      {
+        text: '',
+        link: '',
+      },
+    ],
+    search: {
+      enabled: true,
+      indexName: 'learn-hasura-backend',
```

Comment:
this should be a different index like `learn-hasura-v3-supergraph` for Algolia index name?